CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 ## 6.9.0 (Unreleased)
 
+NOTES:
+* provider: minor refactoring of how we run acceptance tests in VCR mode ([#19879](https://github.com/hashicorp/terraform-provider-google/pull/19879))
+
```

Comment:
We might want to remove this, but figured I'd flag for others to give their opinion on.

```suggestion
```

Reply:
I agree. I don't think this change is user-facing enough to need a release note.